tests/python/pants_test/engine/legacy/test_graph.py

```diff
@@ -32,164 +21,59 @@ def macro(target_cls, tag, parse_context, tags=None, **kwargs):
-        self.create_graph_from_specs(graph_helper, ['no-such-path:'])
+      self.targets('no-such-path:')
 
     self.assertIn('Path "no-such-path" does not contain any BUILD files',
                   str(cm.exception))
 
-  def test_with_existing_directory_with_no_build_files_fails(self):
-    with self.assertRaises(AddressLookupError) as cm:
-      path_ignore_patterns=[
-        # This is a symlink that points out of the build root.
-        '/build-support/bin/native/src'
-      ]
-      with self.graph_helper(path_ignore_patterns=path_ignore_patterns) as graph_helper:
-        self.create_graph_from_specs(graph_helper, ['build-support/bin::'])
-
-    self.assertIn('does not match any targets.', str(cm.exception))
-
-  def test_inject_bad_dir(self):
-    with self.assertRaises(AddressLookupError) as cm:
-      with self.graph_helper() as graph_helper:
-        graph, target_roots = self.create_graph_from_specs(graph_helper, ['3rdparty/python:'])
-
-        graph.inject_address_closure(Address('build-support/bin', 'wat'))
-
-    self.assertIn('Path "build-support/bin" does not contain any BUILD files',
-                  str(cm.exception))
-
-
-class GraphInvalidationTest(GraphTestBase):
-
   def test_invalidate_fsnode(self):
     # NB: Invalidation is now more directly tested in unit tests in the `graph` crate.
-    with self.open_scheduler(['3rdparty/python::']) as (_, _, scheduler):
-      invalidated_count = scheduler.invalidate_files(['3rdparty/python/BUILD'])
-      self.assertGreater(invalidated_count, 0)
+    self.create_library('src/example', 'target', 'things')
+    self.targets('src/example::')
+    invalidated_count = self.invalidate_for('src/example/BUILD')
+    self.assertGreater(invalidated_count, 0)
 
-  def test_invalidate_fsnode_incremental(self):
-    # NB: Invalidation is now more directly tested in unit tests in the `graph` crate.
-    with self.open_scheduler(['//:', '3rdparty/::']) as (_, _, scheduler):
-      # Invalidate the '3rdparty/python' DirectoryListing, the `3rdparty` DirectoryListing,
-      # and then the root DirectoryListing by "touching" files/dirs.
-      for filename in ('3rdparty/python/BUILD', '3rdparty/jvm', 'non_existing_file'):
-        invalidated_count = scheduler.invalidate_files([filename])
-        self.assertGreater(invalidated_count,
-                           0,
-                           'File {} did not invalidate any Nodes.'.format(filename))
-
-  def _ordering_test(self, spec, expected_sources=None):
-    expected_sources = expected_sources or ['p', 'a', 'n', 't', 's', 'b', 'u', 'i', 'l', 'd']
-    with self.open_scheduler([spec]) as (graph, _, _):
-      target = graph.get_target(Address.parse(spec))
-      sources = [os.path.basename(s) for s in target.sources_relative_to_buildroot()]
-      self.assertEqual(expected_sources, sources)
-
-  def test_sources_ordering_literal(self):
-    self._ordering_test('testprojects/src/resources/org/pantsbuild/testproject/ordering:literal')
```

Comment:
Thankfully it looks like the last remaining reference is testprojects/src/resources/org/pantsbuild/testproject/buildfile_path/BUILD and that can be nuked outright.